client/views/root/canada_post.js

```diff
@@ -0,0 +1,3 @@
+import RootView from '../generic-settings-form';
```

Comment:
So far (and we intend on keeping it that way), there is no difference between USPS and CanadaPost in the eye of the plugin.